db/schema.rb

```diff
@@ -197,6 +197,25 @@
 
   add_index "faqs", ["category"], name: "index_faqs_on_category", using: :btree
 
+  create_table "group_users", force: :cascade do |t|
+    t.integer  "user_id",    limit: 4
+    t.integer  "group_id",   limit: 4
```

Comment:
As a rule, whenever you have a foreign key in a table, consider adding an index on that key.

http://stackoverflow.com/questions/3650690/should-every-sql-server-foreign-key-have-a-matching-index